enterprise/j2ee.ddloaders/src/org/netbeans/modules/j2ee/ddloaders/multiview/CmpFieldsNode.java

```diff
@@ -89,7 +89,7 @@ public SectionNode getNodeForElement(Object element) {
             }
         } else if (element instanceof CmpField[]) {
             final List list1 = Arrays.asList(cmpFields.getCmpFields());
-            final List list2 = new LinkedList(Arrays.asList((CmpField[]) element));
+            final List list2 = new LinkedList<>(Arrays.asList((CmpField[]) element));
```

Comment:
hi brad. I noticed a few declarations don't have a generic type on the left hand side. Was it not easily possible to assign one without breaking things?

Reply:
Hey @mbien , you are correct. I spent most of the time really trying to make the actual variables into a generic type and it was just not possible without also changing the public interfaces. Eventually, I'll get there, just not today. 8-)